templates/vue-demo-store/components/account/AccountMenu.vue

```diff
@@ -39,12 +45,11 @@ async function invokeLogout() {
           >
             <div class="flex items-center justify-end">
               <div class="text-sm">
-                <NuxtLink
-                  :to="formatLink(`/account/recover`)"
-                  class="font-medium text-indigo-600 hover:text-indigo-500"
+                <span
```

Comment:
this changes routes, so should stay link

if click event is really needed than it should be a button for accessibility

maybe some router hook here to close it on navigation?

Reply:
but you also can attach additional `@click` for link and that would solve both problems

Reply:
done